src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/MobilePreviewEditModePageObject.java

```diff
@@ -0,0 +1,103 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.article.editmode;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MobilePreviewEditModePageObject extends BasePageObject {
+
+  @FindBy(css = ".mobile-preview")
+  private WebElement previewModal;
+
+  @FindBy(css = ".mobile-preview iframe")
+  private WebElement mobilePreviewIframe;
+
+  @FindBy(css = ".portable-infobox")
+  private WebElement infobox;
+
+  @FindBy(css = ".portable-infobox .pi-title")
+  private WebElement infoboxTitle;
+
+  @FindBy(css = ".portable-infobox .pi-hero")
+  private WebElement infoboxHeroImage;
+
+  @FindBy(css = ".portable-infobox .pi-data")
+  private WebElement infoboxData;
+
+  @FindBy(css = ".article-content p")
+  private WebElement articleContent;
+
+  @FindBy(css = ".article-content .article-video")
+  private WebElement articleVideo;
+
+  @FindBy(css = ".article-content .article-image img")
+  private WebElement articleImage;
+
+  @FindBy(css = ".article-content .article-image figcaption")
+  private WebElement articleImageCaption;
+
+  @FindBy(css = ".ember-container .article-body")
+  private WebElement previewArticleBody;
+
+  public MobilePreviewEditModePageObject() {
+    super();
+  }
+
+  public MobilePreviewEditModePageObject openMobilePreview(String articleName) {
+    EditMode editMode = new ArticlePageObject().open(articleName).navigateToArticleEditPage();
+    editMode.mobilePreviewArticle();
+    wait.forElementVisible(previewModal);
+    driver.switchTo().frame(mobilePreviewIframe);
+
+    return this;
+  }
+
+  public boolean isInfoboxDisplayed() {
+    wait.forElementVisible(infobox);
```

Comment:
this will throw timeout if `infobox` is not visible.
use `wait.forElementPresent()` instead, but you will have to create `By` variable from that selector